### PR TITLE
open-cluster-management/backlog#4770 peer review whole topic

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

File: manage_applications/subscribe_git_resources.adoc [hunk 1]
```diff
@@ -12,20 +12,20 @@ Learn how to grant subscription administrator access.
...
+. From the terminal, log in to your {product-title-short} cluster.
 
 . Add the following subjects into `open-cluster-management:subscription-admin` ClusterRoleBinding with the following command:
 +
 ----
 oc edit clusterrolebinding open-cluster-management:subscription-admin
 ----
 +
-*Note:* `open-cluster-management:subscription-admin` ClusterRoleBinding has no subject initially.
+*Note:* `open-cluster-management:subscription-admin` initially ClusterRoleBinding has no subject.
```
Comment (cadawson) [inline manage_applications/subscribe_git_resources.adoc]: This looked like a sentence before, but not now. Should "initially" be capitalized and have a comma following it?

File: manage_applications/subscribe_git_resources.adoc [hunk 2]
```diff
@@ -12,20 +12,20 @@ Learn how to grant subscription administrator access.
 
 . From the console, log in to your {ocp-short} cluster.
```
Comment (cadawson) [inline manage_applications/subscribe_git_resources.adoc]: Use {ocp} on the first occurrence.

**Review by cadawson — COMMENTED** [review]
Just a couple of quick comments.

File: manage_applications/subscribe_git_resources.adoc (hunk 1 above)
Comment (dockerymick) [reply manage_applications/subscribe_git_resources.adoc]: I agree here, I would change this to:

Initially, `open-cluster-management:subscription-admin` ClusterRoleBinding has no subject.

Comment (dockerymick) [reply manage_applications/subscribe_git_resources.adoc]: The `+` sign for alignment is also visible when i switch to the preview tab

**Review by dockerymick — CHANGES_REQUESTED** [review]
I think you can remove the plus signs in the `Resource overwrite example` section. The codeblocks are not aligning well. The alignment looks fine in the section before that tho

File: manage_applications/subscribe_git_resources.adoc (hunk 1 above)
Comment (swopebe) [reply manage_applications/subscribe_git_resources.adoc]: Yes, I saw that in the portal a couple of times--I am thinking it doesn't like bold after it but I have to investigate that one.

File: manage_applications/subscribe_git_resources.adoc (hunk 2 above)
Comment (swopebe) [reply manage_applications/subscribe_git_resources.adoc]: good point

**Comment by swopebe** [comment]
didn't even see those, removed @dockerymick

**Comment by swopebe** [comment]
gonna merge so Roke can close within sprint

**Pull request merged**